Description: Add git files
 The archlinux submodule is not included in the Ubuntu source.
 .
 waybar (0.10.3-1ppa1) noble; urgency=medium
 .
   * Repackage for noble.
Author: Constantin Piber <cp.piber@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-05-31

--- /dev/null
+++ waybar-0.10.3/package/archlinux/.SRCINFO
@@ -0,0 +1,22 @@
+pkgbase = waybar-git
+	pkgdesc = Highly customizable wayland bar for sway or wlroots based compositor.
+	pkgver = r32.03132bd2
+	pkgrel = 1
+	url = https://github.com/Alexays/Waybar/
+	arch = x86_64
+	license = MIT
+	makedepends = meson
+	depends = gtkmm3
+	depends = jsoncpp
+	depends = libinput
+	depends = libnl>=3.0
+	depends = libsigc++>=2.0
+	depends = wayland
+	depends = wlroots
+	source = waybar-git::git://github.com/Alexays/Waybar
+	source = meson.build.patch
+	sha1sums = SKIP
+	sha1sums = 36bd667affd6f5e3ca3e15665f7af2ea8438eda7
+
+pkgname = waybar-git
+
--- /dev/null
+++ waybar-0.10.3/package/archlinux/PKGBUILD
@@ -0,0 +1,37 @@
+# Maintainer: Alexis Rouillard <contact@arouillard.fr>
+
+pkgname=waybar-git
+pkgver=r32.03132bd2
+pkgrel=1
+pkgdesc='Highly customizable wayland bar for sway or wlroots based compositor.'
+arch=('x86_64')
+url='https://github.com/Alexays/Waybar/'
+license=('MIT')
+depends=('gtkmm3'
+         'jsoncpp'
+         'libinput'
+         'libnl>=3.0'
+         'libsigc++>=2.0'
+         'wayland'
+         'wlroots')
+makedepends=('meson')
+source=("${pkgname}::git://github.com/Alexays/Waybar"
+        'meson.build.patch')
+sha1sums=('SKIP'
+          '36bd667affd6f5e3ca3e15665f7af2ea8438eda7')
+
+prepare() {
+    cd "${srcdir}/${pkgname}"
+    patch meson.build "${srcdir}/meson.build.patch"
+}
+
+build() {
+    cd "${srcdir}/${pkgname}"
+    rm -rf "${srcdir}/build"
+    meson --wrap-mode=nofallback --prefix=/usr "${srcdir}/build"
+    ninja -C "${srcdir}/build"
+}
+
+package() {
+    DESTDIR="$pkgdir" ninja -C "${srcdir}/build" install
+}
--- /dev/null
+++ waybar-0.10.3/package/archlinux/meson.build.patch
@@ -0,0 +1,8 @@
+1c1
+< project('waybar', 'cpp', 'c', default_options : ['cpp_std=c++17'])
+---
+> project('waybar', 'cpp', 'c', default_options : ['cpp_std=c++17', 'default_library=static'])
+21c21
+< fmt = dependency('fmt', fallback: ['fmtlib', 'fmt_dep'])
+---
+> fmt = subproject('fmtlib').get_variable('fmt_dep')
